### PR TITLE
SEO 3

### PR DESCRIPTION
**Pull request by sekrystal**
- Reformatted “about” page to effective metatag syntax
- Lengthened “home” and “playground” page meta-descriptions for visibility

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -19,14 +21,17 @@ const SectionWrapper: React.FC<SectionWrapperProps> = ({
...
-          name="description"
-          content="Demystifying what makes the Ethereum Virtual Machine work."
-        />
-      </Head>
+      <html lang="en"></html>
+      <React.Fragment>
+        <Head>
+          <title> EVM Codes - About the EVM </title>
+          <h1>
+            Understanding the Ethereum Virtual Machine (or EVM) is understanding
```
Comment (dorlevi) [inline pages/about.tsx:29]: I don't love this language, it's too much like a fortune cookie a proposal "Understand the relationship between opcode instructions, gas costs, storage, and the execution environment to better understand the EVM" or something

File: pages/playground.tsx [hunk 2]
```diff
@@ -8,9 +8,10 @@ import { Container } from 'components/ui'
 const PlaygroundPage = () => {
   return (
     <>
+      <html lang="en"></html>
       <Head>
         <meta property="og:type" content="website" />
-        <meta property="title" content="EVM Codes - Playground" />
+        <title>EVM Codes - Playground</title>
```
Comment (dorlevi) [inline pages/playground.tsx:14]: You don't want to add something here EVM Codes - Explore your contracts in our Playground" or something?

Comment (sekrystal) [reply pages/playground.tsx:14]: That'd be too long – the longest tagline we'd want is for the homepage. This particular text is what you'd see in a tab bar, the shorter, the better.

**Review by dorlevi — APPROVED** [review]
LGTM

**Pull request merged**